docs/sources/panels/panel-editor.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Panel editor"
+type = "docs"
+[menu.docs]
+identifier = "panel-editor"
+weight = 200
++++
+
+# Panel editor
+
+This page describes the parts of the Grafana panel editor and links to where you can find more information.
+
+{{< docs-imagebox img="/img/docs/panel-editor/panel-editor-7-0.png" class="docs-image--no-shadow" max-width="1500px" >}}
+
+## Open the Panel editor
+
+There are several ways to access the panel editor, also called the **Edit Panel** screen, _edit mode_, or _panel edit mode_:
+
+- Click the **Add panel** icon at the top of the screen and then click **Add new panel**. The new panel opens in the panel editor. For detailed instructions on how to add a panel, refer to [Add a panel]({{< relref "add-a-panel.md" >}})
+- Click the title of an existing panel and then click **Edit**. The panel opens in edit mode.
+- Click anywhere on an existing panel and then press **e** on your keyboard. The panel opens in edit mode.
+
+## Resize panel editor sections
+
+Drag to resize sections of the panel editor. If the Panel, Field, and Overrides tabs become too narrow, then the tabs convert to a dropdown list.
+
+{{< docs-imagebox img="/img/docs/panel-editor/resize-panel-editor-panels-7-0.gif" class="docs-image--no-shadow" max-width="600px" >}}
+
+## Parts of the panel editor
+
+This section describes the parts of the panel editor screen and a bit about fields, options, or tasks associated with each part. Some sections in this page link to pages where sections or tasks are documented more fully.
+
+### Header
+
+The header section lists the name of the dashboard that the panel is in and some dashboard commands. You can also click the **Go back** arrow to return to the dashboard.
+
+{{< docs-imagebox img="/img/docs/panel-editor/edit-panel-header-7-0.png" class="docs-image--no-shadow" max-width="1000px" >}}
+
+On the right side of the header are the following options:
+
+- **Dashboard settings (gear) icon -** Click to access the dashboard settings.
+- **Discard -** Discards all changes you have made since you last saved the dashboard.
+- **Save -** Saves the dashboard, including all changes you have made in the panel editor.
+- **Apply -** Applies changes you made and then closes the panel editor, returning you to the dashboard.
+
+Panel title, access dashboard settings, Save/Apply/Discard
+
+### Visualization preview
+
+The visualization preview section contains viewing options, time range controls, the visualization preview, and (if applicable) the panel title, axes, and legend.
+
+{{< docs-imagebox img="/img/docs/panel-editor/visualization-preview-7-0.png" class="docs-image--no-shadow" max-width="1200px" >}}
+
+- **Fill -**
+- **Fit -**
+- **Exact -**
+- **Time range controls -** For more information, refer to [Time range controls](ADD LINK WHEN TOPIC IS MERGED).
+
+### Data section
+
+The section contains tabs where you enter queries, transform your data, and create alert rules.
```

Comment:
```suggestion
The section contains tabs where you enter queries, transform your data, and create alert rules (if applicable).
```

Reply:
It's not always possible to create alert rules. Eg. use of transformations block this.